### PR TITLE
Added a term entry for is-nan in PHP (#2087)

### PR DESCRIPTION
**Pull request by tipofey**
### Description
Added a new term entry for **PHP** about `is_nan` function.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+Title: 'is-nan'
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:2]:
```suggestion
Title: 'is-nan()'
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+Title: 'is-nan' 
+Description: 'Finds whether a value is not a number.'
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:3]:
```suggestion
Description: 'Returns a boolean indicating if a value is not a number.'
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md (hunk 1 above)
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:2]:
```suggestion
Title: 'is-nan()'
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'An nth tag'
+CatalogContent: 
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`is_nan`** function checks whether the argument is 'not a number'.
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:17]:
```suggestion
The **`is_nan()`** function checks whether the argument is not a number.
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Syntax
+
+```pseudo
+$result = is_nan(n);
+```
+
+Returns **true** if `n` is 'not a number', else **false**.
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:25]:
```suggestion
Returns `true` if `n` is 'not a number', else `false`.
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Codebyte Example
+
+Using `is_nan()` to check whether `10` is 'not a number':
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:29]:
```suggestion
Using `is_nan()` to check whether `10` is not a number:
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+```codebyte/php
+<?php
+  echo(is_nan(10));
+?>
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:34]:
```suggestion
?>
```
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+Title: 'is-nan' 
+Description: 'Finds whether a value is not a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'An nth tag'
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:11]:
```suggestion
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md (hunk 4 above)
Comment (yangc95) [reply content/php/concepts/math-functions/terms/is-nan/is-nan.md:25]: It looks like when it's false, it returns nothing and returns 1 when true

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md (hunk 6 above)
Comment (yangc95) [reply content/php/concepts/math-functions/terms/is-nan/is-nan.md:34]: Nothin should be echoed because it's false

**Review by yangc95 — COMMENTED** [review]
@tipofey nice succinct entry 👍 I left a few comments and suggestions. Please get to them when you have a chance!

**Comment by tipofey** [comment]
Cheers @yangc95 ! Not sure it was the most efficient way, but I've created a new pull request based on your suggestions.

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+Returns `1` if `n` is not a number, else returns nothing.
+
+## Codebyte Example
+
+Using `is_nan()` to check whether `10` is 'not a number':
+
+```codebyte/php
+<?php
+echo "10:", is_nan(10) . "<br>";
+echo "acos 1.01: ", is_nan(acos(1.01));
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:34]:
```suggestion
  echo "10:", is_nan(10), "\n";
  echo "sqrt of -1: ", is_nan(sqrt(-1));
```

Comment (yangc95) [reply content/php/concepts/math-functions/terms/is-nan/is-nan.md:34]: I like this example, since sort of a negative number is undefined

**Comment by yangc95** [comment]
@tipofey I don't see a new pull request. It looks like you correctly made changes on this PR which is great.

I left one suggestion on this PR, after which I can approve and we can get a second and final reviewer on this 🙂 thanks!

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
+---
+Title: 'is-nan()'
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:2]:
```suggestion
Title: 'is_nan()'
```

File: content/php/concepts/math-functions/terms/is-nan/is-nan.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+Title: 'is-nan()' 
+Description: 'Checks whether a value is not a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/is-nan/is-nan.md:11]:
```suggestion
```

**Review by yangc95 — APPROVED** [review]
@tipofey With the exception of the two required suggestions I left in the metadata, this looks good for a second reviewer 👍

**Review by KTom101 — CHANGES_REQUESTED** [review]
@tipofey I took a look at this and have one change, thanks.

**Comment by tipofey** [comment]
@KTom101 Thanks! Is there anything I need to do at this point?

**Review by KTom101 — APPROVED** [review]
@tipofey No more you have to do. I went ahead and ran `npm run format:all` since the tests were failing. You can read more about that [here](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md#how-can-i-test-my-changes-before-submitting-a-pr) for future entries.

I will be merging this in a minute.

**Comment by KTom101** [comment]
The link is live: https://www.codecademy.com/resources/docs/php/math-functions/is-nan

You're currently listed as anonymous:

Screenshot 2023-05-15 at 4 47 23 PM

If you want to remain anonymous, there is nothing more you need to do. If you wish to be listed as a contributor, make sure you've added your GitHub username to your Codecademy profile using the following steps:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

> **Note:** Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

@yangc95 @SSwiniarski

**Pull request merged**